Makefile-test.mk

```
@@ -103,24 +103,30 @@ test-multikueue-integration: gomod-download envtest ginkgo dep-crds kueuectl gin
 	$(PROJECT_DIR)/bin/ginkgo-top -i $(ARTIFACTS)/multikueue-integration.json > $(ARTIFACTS)/multikueue-integration-top.yaml
 
 CREATE_KIND_CLUSTER ?= true
+
+SINGLECLUSTER-E2E_TARGETS := $(addprefix run-test-e2e-singlecluster-,$(E2E_KIND_VERSION:kindest/node:v%=%))
+CUSTOMCONFIGS-E2E_TARGETS := $(addprefix run-test-customconfigs-e2e-,$(E2E_KIND_VERSION:kindest/node:v%=%))
```

```suggestion
SINGLECLUSTER-E2E_TARGETS := $(addprefix run-test-e2e-singlecluster-,$(E2E_KIND_VERSION:kindest/node:v%=%))
CUSTOMCONFIGS-E2E_TARGETS := $(addprefix run-test-e2e-customconfigs-,$(E2E_KIND_VERSION:kindest/node:v%=%))
```
nit, to make the naming consistent. I see multikueue names it different, but I would suggest to adjust the multikueue target name in a follow up. It seems reasonable that e2e is before "customconfigs" as a more generic term. It is also consistent with the dir structure.